doc/bottomSheet.md

```diff
@@ -0,0 +1,135 @@
+# BottomSheet
+
+Mística provides a bottom sheet component that can be used to display a modal-like content from over the main
+content of the screen.
+
+## Basic usage
+
+You can show any content you want inside the bottom sheet by passing it as a child of the component.
+
+```jsx
+import {BottomSheet} from 'mistica';
+
+const MyComponent = () => {
+  const [showSheet, setShowSheet] = useState(false);
+  return (
+    <>
+      <ButtonPrimary onPress={() => setShowSheet(true)}>show bottom sheet</ButtonPrimary>
+      {showSheet && (
+        <BottomSheet onClose={() => setShowSheet(false)}>
+          <Placeholder />
+        </BottomSheet>
+      )}
+    </>
+  );
+};
+```
+
+The sheet will close when the user does the swipe down gesture or when the background overlay is touched. The
+`onClose` callback is called when the closing animation finishes, that's the right place to unmount the sheet
+as shown in the example above.
+
+You can also close the sheet programmatically using the render prop:
+
+```jsx
+import {BottomSheet} from 'mistica';
+
+const MyComponent = () => {
+  const [showSheet, setShowSheet] = useState(false);
+  return (
+    <>
+      <ButtonPrimary onPress={() => setShowSheet(true)}>show bottom sheet</ButtonPrimary>
+      {showSheet && (
+        <BottomSheet onClose={() => setShowSheet(false)}>
```

Comment:
note that `onClose` is called when the closing animation finishes, so the idea is not unmounting the component until the transition finishes, that's why I set showSheet to false in onClose handler instead of ButtonPrimary onPress handler